cmd/debos/debos.go

```diff
@@ -15,18 +15,18 @@ import (
 	"github.com/jessevdk/go-flags"
 )
 
-func checkError(context *debos.DebosContext, err error, a debos.Action, stage string) int {
+func checkError(context *debos.DebosContext, err error, a debos.Action, stage string) bool {
 	if err == nil {
-		return 0
+		return true
 	}
 
 	context.State = debos.Failed
 	log.Printf("Action `%s` failed at stage %s, error: %s", a, stage, err)
 	debos.DebugShell(*context)
```

Comment:
Side nit: is it only me that finds the random call to "DebugShell" in a function call "checkError" odd?

Reply:
right, the idea here is that if there is an error, instead of quitting we show a debug shell if that's what the user wanted. I'd be happy to accept a PR to add a comment to do that if you like.

Reply:
actually, I think I can handle that inside this PR.